docs/release-notes/2.14.0.md

```diff
@@ -0,0 +1,77 @@
+# Percona Monitoring and Management 2.14.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">January 26, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Release Highlights
+
+- Switch to push metrics by default
+
+    In PMM v2.12.0, Percona replaced its metrics collection service (formerly Prometheus) with VictoriaMetrics. Historically, PMM used a "pull" method with Prometheus. VictoriaMetrics can operate in either a "pull" or "push" method. When PMM v2.12.0 was released, Percona kept the method as "pull". Now with PMM v2.14.0, Percona is shifting the default to "push". This [blog post](https://www.percona.com/blog/2020/12/01/foiled-by-the-firewall-a-tale-of-transition-from-prometheus-to-victoriametrics/) describes the two methods and why "push" benefits its users. Also, [here is a post](https://www.percona.com/blog/2020/12/16/percona-monitoring-and-management-migration-from-prometheus-to-victoriametrics-faq/) by Peter Zaitzev of FAQs relating to the move to VictoriaMetrics and the "push" model. Documentation on the "push" method is [here](../details/victoria-metrics.md).
```

Comment:
```suggestion
    In PMM v2.12.0, Percona replaced its metrics collection service (formerly Prometheus) with VictoriaMetrics. Historically, PMM used a "pull" method with Prometheus while VictoriaMetrics can operate in either a "pull" or "push" method. When PMM v2.12.0 was released, Percona kept the default method as "pull". Now with PMM v2.14.0, Percona is shifting the default to "push" for all newly added instances. This [blog post](https://www.percona.com/blog/2020/12/01/foiled-by-the-firewall-a-tale-of-transition-from-prometheus-to-victoriametrics/) describes the two methods and why "push" benefits its users. Also, [here is a post](https://www.percona.com/blog/2020/12/16/percona-monitoring-and-management-migration-from-prometheus-to-victoriametrics-faq/) by Peter Zaitzev of FAQs relating to the move to VictoriaMetrics and the "push" model. Documentation on the "push" method is [here](../details/victoria-metrics.md).
```

Reply:
Change made in https://jira.percona.com/browse/RM-884 ready for next jirabot run